### DISCUSSION TITLE
Where is it?

### DISCUSSION THREAD

**Discussion by cloudpaul** [GSI-CS-CO/chart-fx#590] [General]
The README states that this library is deployed to Maven central, but I cannot find anything for groupId **io.fair-acc**.
So, where is it please? 
Version 11.3.0 that is
Thanks!

**Comment by wirew0rm** [comment]
11.3.0 which will move to the new coordinates is not yet released, you can either use the snapshot version or the old coordinates as described in https://github.com/fair-acc/chart-fx/blob/11.2.7/README.md#add-the-library-to-your-project

Sorry for the confusion, I was too optimistic with being able to release the new version when updating the README :/ Hope this helps :)

**Comment by cloudpaul** [comment]
Thanks for your reply
